jobs/build-node-image.Jenkinsfile

```diff
@@ -134,12 +148,12 @@ lock(resource: "build-node-image") {
         throw e
     } finally {
         if (currentBuild.result == 'SUCCESS') {
-            currentBuild.description = "[${params.RELEASE}@${shortcommit}] ⚡"
+            currentBuild.description = "[${build_description}@${shortcommit}] ⚡"
         } else {
-            currentBuild.description = "[${params.RELEASE}@${shortcommit}] ❌"
+            currentBuild.description = "[${build_description}@${shortcommit}] ❌"
         }
         if (currentBuild.result != 'SUCCESS') {
-            message = ":openshift: build-node-image #${env.BUILD_NUMBER} <${env.BUILD_URL}|:jenkins:> <${env.RUN_DISPLAY_URL}|:ocean:> [${params.RELEASE}][${src_config_ref}@${shortcommit}]"
+            message = ":openshift: build-node-image #${env.BUILD_NUMBER} <${env.BUILD_URL}|:jenkins:> <${env.RUN_DISPLAY_URL}|:ocean:> ${build_description}"
```

Comment:
This loses the git ref and shortcommit info.

I think instead the way I would structure this is that we keep all the info in `build_description`, and then whenever we update `currentBuild.description`, it's only to append the "state" part of it. I.e. [near the start](https://github.com/jbtrystram/fedora-coreos-pipeline/blob/43cfa03d6c776e3e49e1f5e243463f5ef007f355/jobs/build-node-image.Jenkinsfile#L63):

```
currentBuild.description = "${build_description} Running"
```

and then at the beginning of the `finally` just above

```
currentBuild.description = "${build_description} ⚡"
```

and then here we can just use `build_description` only as you're doing.